utils/constants.ts

```diff
@@ -14,6 +14,13 @@ export const UNLIMITED_ALLOWANCE_IN_BASE_UNITS = new BigNumber(2).pow(256).minus
 export const MAX_DIGITS_IN_UNSIGNED_256_INT = 78;
 export const ZRX_TOKEN_ADDRESS = '0x871dd7c2b4b25e1aa18728e9d5f2af4c4e431f5c';
 
+// Rebalancing Constants
+export const DEFAULT_PERIOD_INTERVAL = new BigNumber(90000);
```

Comment:
Helps to be even more realistic here for clarity. Your contracts specify day cutoffs. Create constants for maybe `ONE_DAY_IN_SECONDS` and your test case for invalid `PERIOD_INTERVAL` can be `ONE_DAY_IN_SECONDS.sub(1)`

Reply:
It makes your usage more readable: `const proposalPeriod = ONE_DAY_IN_SECONDS;` vs.
`const proposalPeriod = DEFAULT_PERIOD_INTERVAL;`